stablehlo/integrations/python/mlir/dialects/stablehlo.py

```diff
@@ -17,3 +17,11 @@
 # pylint: disable=wildcard-import,relative-beyond-top-level,g-import-not-at-top
 from ._stablehlo_ops_gen import *
 from .._mlir_libs._stablehlo import *
+
+
+def get_earliest_forward_compatible_version():
+  """Return the earliest StableHLO version that the current StableHLO version
+
+  is still forward compatible with.
+  """
+  return "0.9.0"
```

Comment:
Should we also expose this API through `stablehlo-translate`?

Reply:
Is there any reason for that? We don't expose any other language bindings or C/Python APIs via `stablehlo-translate`? If we were to expose it, I would think we should do it the same way we do `--target=current`. Something like `--target=earliest` or `minimum`